### PR TITLE
docs: clarification on the naming

### PR DESCRIPTION
**Pull request by mfranzke**
- It's unclear what the `DB UX System` would be – we should either write `DB UX Design System` if we mean the Design System or add the appendix, if we want to add a link to the technical components part.
- Corrected some path

### CODE REVIEW

**Comment by mfranzke** [comment]
@nmerget it's unclear, what this reference would mean – at least the file listed here doesn't exist in the package:
https://github.com/db-ui/mono/pull/329/files#diff-6b6e5a708eeaa016a2c50d3a666cbe6deb9805b367de533f8ba5d0938556b3eaR44

**Comment by nmerget** [comment]
> @nmerget it's unclear, what this reference would mean – at least the file (`@use "@db-ui/react-components/dist/db-ui-components" as *;`) listed here doesn't exist in the [package](https://www.npmjs.com/package/@db-ui/react-components?activeTab=explore): https://github.com/db-ui/mono/pull/329/files#diff-6b6e5a708eeaa016a2c50d3a666cbe6deb9805b367de533f8ba5d0938556b3eaR44

It's old we removed the styles from the framework packages. We just need foundations and/or components for styles. Maybe we should update the examples as well

File: output/react/README.md [hunk 1]
```diff
@@ -41,7 +41,6 @@ import "@db-ui/components/src/styles/db-ui-42-rollup.css";
 ```scss
 @use "@db-ui/foundations/build/scss/rollup.assets-paths" as *;
 @use "@db-ui/foundations/build/scss/icon/icons" as *;
-@use "@db-ui/react-components/dist/db-ui-components" as *;
```
Comment (nmerget) [inline output/react/README.md:44]: We should use `@db-ui/foundations/build/scss/db-ui-foundations` here + all components we want to use with:

`@db-ui/components/build/components/button/button`
`@db-ui/components/build/components/icon/icon`
...

Comment (mfranzke) [reply output/react/README.md:44]: ah, got it, I misunderstood our conversation. Thanks for mentioning this aspect.

Comment (mfranzke) [reply output/react/README.md:44]: @nmerget same for previous `@db-ui/react-components/dist/db-ui-components.css` references with `@db-ui/foundations/build/css/db-ui-foundations.css` than, isn't it?

**Review by nmerget — APPROVED** [review]

**Pull request merged**